### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 0: invalid continuation byte

### ISSUE THREAD

**Issue by mfouesneau** [kivy/pyjnius#183]
At the first import it bugs. Installation did not show any special issue.
## In [1]: from jnius import autoclass

UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-1-28e3c8425f25> in <module>()
----> 1 from jnius import autoclass

/Users/fouesneau/anaconda/lib/python3.4/site-packages/jnius-1.1.dev0-py3.4-macosx-10.5-x86_64.egg/jnius/**init**.py in <module>()
     11 
     12 from .jnius import *
---> 13 from .reflect import *
     14 
     15 # XXX monkey patch methods that cannot be in cython.

/Users/fouesneau/anaconda/lib/python3.4/site-packages/jnius-1.1.dev0-py3.4-macosx-10.5-x86_64.egg/jnius/reflect.py in <module>()
     12 
     13 
---> 14 class Class(with_metaclass(MetaJavaClass, JavaClass)):
     15     **javaclass** = 'java/lang/Class'
     16 

/Users/fouesneau/anaconda/lib/python3.4/site-packages/six.py in **new**(cls, name, this_bases, d)
    776     class metaclass(meta):
    777         def **new**(cls, name, this_bases, d):
--> 778             return meta(name, bases, d)
    779     return type.**new**(metaclass, 'temporary_class', (), {})
    780 

jnius/jnius_export_class.pxi in jnius.MetaJavaClass.**new** (jnius/jnius.c:15553)()

jnius/jnius_export_class.pxi in jnius.MetaJavaClass.resolve_class (jnius/jnius.c:15941)()

jnius/jnius_env.pxi in jnius.get_jnienv (jnius/jnius.c:3248)()

jnius/jnius_jvm_dlopen.pxi in jnius.get_platform_jnienv (jnius/jnius.c:3182)()

jnius/jnius_jvm_dlopen.pxi in jnius.create_jnienv (jnius/jnius.c:2665)()

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 0: invalid continuation byte
##

**Comment by NSMarkop** [comment]
I get the same issue with Python 3.4.3 on Ubuntu 14.04 using the latest 1.1-dev. It works fine with Python 2.7.6 and I have Cython 0.21 installed.

```bash
nick ~/temp $ python3 test.py 
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from jnius import autoclass
  File "/usr/local/lib/python3.4/dist-packages/jnius/__init__.py", line 13, in <module>
    from .reflect import *
  File "/usr/local/lib/python3.4/dist-packages/jnius/reflect.py", line 14, in <module>
    class Class(with_metaclass(MetaJavaClass, JavaClass)):
  File "/usr/local/lib/python3.4/dist-packages/six.py", line 808, in __new__
    return meta(name, bases, d)
  File "jnius_export_class.pxi", line 43, in jnius.MetaJavaClass.__new__ (jnius/jnius.c:15707)
  File "jnius_export_class.pxi", line 65, in jnius.MetaJavaClass.resolve_class (jnius/jnius.c:16095)
  File "jnius_env.pxi", line 11, in jnius.get_jnienv (jnius/jnius.c:3220)
  File "jnius_jvm_dlopen.pxi", line 89, in jnius.get_platform_jnienv (jnius/jnius.c:3154)
  File "jnius_jvm_dlopen.pxi", line 52, in jnius.create_jnienv (jnius/jnius.c:2639)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 0: invalid continuation byte
```

test.py above is just the following code:

```python
from jnius import autoclass

Stack = autoclass('java.util.Stack')
stack = Stack()
stack.push('hello')
stack.push('world')

print(stack.pop())  # --> 'world'
print(stack.pop())  # --> 'hello'
```

**Comment by mixmastamyk** [comment]
And what is the solution/workaround?

Go back to Python 2.7 ???

**Comment by KeyWeeUsr** [comment]
Check the latest version, it should be fixed after #285.
